cypress/support/fragments/finance/funds/funds.js

```diff
@@ -79,6 +83,30 @@ export default {
       .and('have.text', fundCode.concat('-', fiscalYear));
   },
 
+  checkBudgetQuantity: (quantityValue) => {
+    cy.expect(
+      Section({ id: budgetPaneId }).find(HTML(including('Cash balance: $' + quantityValue.toFixed(2)))).exists()
+    );
+    cy.expect(
+      Section({ id: budgetPaneId }).find(HTML(including('Available balance: $' + quantityValue.toFixed(2)))).exists()
+    );
+  },
+
+  openTransactions: () => {
+    cy.expect(Section({ id: 'information' }).find(KeyValue('Transactions')).exists());
+    // TODO: refactor via using interactors. Simple click() doesn't work, need to find a way to work with child
+    cy.xpath(viewTransactionsLinkXpath).click();
+  },
+
+  checkTransaction: (value, fundCode) => {
+    cy.expect(
```

Comment:
are you sure that extra line breaks is ok in such cases? From my point of view your code will have the same readability in case of one line instead of 3 lines and we will get huge additional space in our windows too

Reply:
Updated